Meta.Tests.ps1

```diff
@@ -432,54 +435,88 @@ Describe 'Common Tests - Validate Example Files' -Tag 'Examples' {
     $examplesPath = Join-Path -Path $moduleRootFilePath -ChildPath 'Examples'
     if (Test-Path -Path $examplesPath)
     {
-
-        ## For Appveyor builds copy the module to the system modules directory so it falls
-        ## in to a PSModulePath folder and is picked up correctly.
+        <#
+            For Appveyor builds copy the module to the system modules directory so it falls in to a PSModulePath folder and is
+            picked up correctly.
+            For a user to run the test, they need to make sure that the module exist in one of the paths in env:PSModulePath, i.e.
+            '%USERPROFILE%\Documents\WindowsPowerShell\Modules'.
+            No copying is done when a user runs the test, becuase that could potentially be destructive.
+        #>
         if ($env:APPVEYOR -eq $true)
         {
-            Copy-item -Path $moduleRootFilePath `
-                      -Destination 'C:\WINDOWS\system32\WindowsPowerShell\v1.0\Modules\' `
-                      -Recurse
+            $powershellModulePath = $env:SystemRoot
+
+            'System32', 'WindowsPowerShell', 'v1.0', 'Modules', $repoName | ForEach-Object {
```

Comment:
I changed it now. :)

I did not complain because I also misunderstood the guideline as _never_ use backslashes in paths 😄 
Maybe we should add to the style guideline that it does not apply to OS specific paths?
https://github.com/PowerShell/DscResources/blob/master/StyleGuidelines.md#no-backslash-in-paths